spec.md

```diff
@@ -676,9 +676,8 @@ message ValidateVolumeCapabilitiesRequest {
   // The API version assumed by the CO. This is a REQUIRED field.
   Version version = 1;
 
-  // The information about the volume to check. This is a REQUIRED
-  // field.
-  VolumeInfo volume_info = 2;
+  // The ID of the volume to check. This is a REQUIRED field.
+  VolumeID volume_id = 2;
```

Comment:
on second thought, why just the ID here? one of the reasons that we return `VolumeInfo` in `ListVolumesResponse` is to facilitate stateless plugin backends. By only supplying the ID here instead of the Info, I think that actually hurts stateless support.

What's the reasoning behind using an ID here instead of Info?